.travis/install.sh

```diff
@@ -50,3 +50,15 @@ pip install -r requirements.d/development.txt
 pip install codecov
 python setup.py --version
 pip install -e .[fuse]
+
+if [ -d "$HOME/.rustup/toolchains" ] && [ -f "$HOME/.cargo/env" ]; then
+    source ~/.cargo/env
+    rustup toolchain install stable # also updates
+    rustup default stable
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source ~/.cargo/env
+fi
+
+cd test-wrapper
+cargo build --release
```

Comment:
Do you also want to put the lines for installing rustup in the script? In some VMs (such as newer versions of Ubuntu and Debian) we might just install rust from the package manager.